### ISSUE TITLE
Publish to deno.land/x

### ISSUE THREAD

**Issue by yuhr** [pelotom/runtypes#342]
`runtypes` is now migrating to Deno-first project. The substantial work to the codebase is already done by #339, so actually this project is already Deno-first, but it's yet to publish to [deno.land/x](https://deno.land/x) due to inactivity of @pelotom on GitHub. I'm now trying to contact with him by email, sending following message:

> The important thing is, runtypes is now going to be a Deno-first project. Being Deno-first doesn't actually necessitates publishing to deno.land/x, but it's highly favored. This needs changes to owner-level settings and collaboration with @sant123 who has already been using `runtypes` name in deno.land/x. Could you take some time to do this yourself, or leave this to me by transferring ownership of pelotom/runtypes to me (yuhr/runtypes) or an organization (runtypes/runtypes) with me being a co-owner?
> 
> To be clear, migrating to a Deno-first project doesn't imply stopping publishing the library to www.npmjs.com. There's a build tool to easily generate npm packages from Deno-first projects, namely [dnt](https://github.com/denoland/dnt), thus it'll still be available to Node.js.

**Comment by sant123** [comment]
Now that [JSR](https://jsr.io/) is out. It would be worth to publich runtypes on the new registry too.

**Comment by yuhr** [comment]
The repository has been successfully moved to a new organization, and I'm now co-owner of this repository. I’d like to use this place to thank pelotom for the quick reply! I'm going to take a look at settings in this weekend.

**Comment by yuhr** [comment]
As expected, setting up the webhooks without contacting support results in an error:

```json
{"success":false,"error":"module name is registered to a different repository"}
```

So I've sent an email to modules@deno.com. [Q&A](https://deno.land/x#Q&A) says “please contact support” but I couldn't find where to do so; instead I found the address in [SO](https://stackoverflow.com/a/75227008/6708445), so I'm unsure if this is the correct way to make such a request, but I see no other promising way out there.

The message I wrote is:

> My case is that I've [got a co-ownership of an OSS called runtypes](https://github.com/runtypes/runtypes/issues/342#issuecomment-1985182010) (whose original owner has been inactive for years), and I'm now going to publish the library to deno.land/x, but there's [unofficial releases](https://deno.land/x/runtypes) tied to [sant123/runtypes](https://github.com/sant123/runtypes) (whose owner is another from the original owner). I already got the [agreement on this with sant123](https://github.com/runtypes/runtypes/issues/137#issuecomment-1586003318), so could you change the associated repository to the [official one](https://github.com/runtypes/runtypes)? The webhook is set up, thus now I think all we need is your manual support to change that.

**Comment by sant123** [comment]
I think the easiest solution for this is to create a scope and package on jsr.io. Since deno.land/x in some moment will stop being the preferred way to publish third party modules, you can start supporting the package in there. Also jsr supports members for publishing so this type of issue won’t be a problem anymore.

**Comment by joeyparrish** [comment]
Is runtypes 7 still blocked on this?  I see some features I would really like to try in the `master` branch README.

**Comment by yuhr** [comment]
There're some remaining bugs not filed as issues, and fixes to them would also result in breaking changes, so I'd like to include them in v7. Thank you for your interest, I'm going to make haste!

**Comment by joeyparrish** [comment]
Is there any info about those bugs? Would it be dumb to start using the code from `main` right now while I'm prototyping a new project?

**Comment by yuhr** [comment]
Bugs as far as I noticed have been all resolved just now! For new projects, the current `master` would be okay to use. Only a few refactoring is left to release.

**Comment by joeyparrish** [comment]
Alternatively, a minor version release that includes https://github.com/runtypes/runtypes/pull/391 would also work for me.  It is `.exact()` that I am most excited to use.  (`.withParser()` second most.)  Not sure if that `.exact()` `feat` depends on any `feat!`s, though.

**Comment by yuhr** [comment]
Sorry we're not providing any backports due to the maintenance cost...

I'm now preparing for a release i.e. testing built packages can be imported correctly in other runtimes such as Node.js and React Native, so please wait a while.

**Comment by yuhr** [comment]
v7 has been released to npmjs.com and jsr.io, but [I still kind of miss some immutable registry with HTTPS imports could happen](https://github.com/jsr-io/jsr/issues/735), so I'm leaving this open until the situation changes, in order to maintain attention for this topic.

**Comment by joeyparrish** [comment]
Thank you so much!  I'm checking it out now.  Looks like .with() will be amazing to use!
